pom.xml

```diff
@@ -180,6 +180,28 @@
 					<artifactId>tycho-p2-repository-plugin</artifactId>
 					<version>${tycho-version}</version>
 				</plugin>
+				<plugin>
+					<groupId>org.eclipse.tycho.extras</groupId>
+					<artifactId>tycho-version-bump-plugin</artifactId>
+					<version>${tycho-version}</version>
+					<executions>
+						<execution>
+							<id>update-target</id>
+							<configuration>
+								<updateEmptyVersion>false</updateEmptyVersion>
```

Comment:
I have currently disabled this as we use the `0.0.0` version in the target, the question for me would be if we can automate this if it is then maybe better to use exact versions in the target.